### ISSUE TITLE
Windows Hyper-V conflicts with Node/Express default port assignment

### ISSUE THREAD

**Issue by c-simpson** [CodeForPhilly/paws-data-pipeline#137]
The client's default port of 3000 is reserved by Windows Hyper-V. `docker-compose up` will get the following error:
```

ERROR: for paws-compose-client  Cannot start service client: Ports are not available: listen tcp 
0.0.0.0:3000: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

ERROR: Encountered errors while bringing up the project.
```

There are three possible solutions:
* Disable Hyper-V and run Docker on WSL2 (Docker's preferred solution)

* Edit docker-compose.yml to expose a different port for the client outside the container, e.g.,  
`
ports:
      - "8000:3000"
`

* Disable Hyper-V, reserve port 3000 so Hyper-V can't do so, re-enable Hyper-V

**Comment by c-simpson** [comment]
I'm happy to write this up as a troubleshooting guide. Would it be better to start a new Troubleshooting page or to add it to `GettingStarted.md` ?

**Comment by urirot** [comment]
@c-simpson maybe as a wiki page in the project and link it in the readme?

**Comment by c-simpson** [comment]
Created new [Troubleshooting page](https://github.com/CodeForPhilly/paws-data-pipeline/wiki/Troubleshooting) in the wiki, will add to Readme.md

**Comment by c-simpson** [comment]
Resolved with PR #145
